source/catalog.html

```diff
@@ -117,6 +121,7 @@ <h3 class="media-preview__caption">Мост дьявола</h3>
       <div class="video-preview__wrapper">
         <h2 class="video-preview__caption-video">Все еще сомневаетесь?</h2>
         <p class="video-preview__subtitle-video">Смотрите видеопрезентацию и скорее за билетами, пока они не подорожали в очередной раз!</p>
+        <div class="video-preview-background"></div>
```

Comment:
```suggestion
        <div class="video-preview__background"></div>
```
Так как это элемент блока `video-preview`.